workspace-hack/Cargo.toml

```diff
@@ -66,14 +66,14 @@ hickory-proto = { version = "0.24.4", features = ["text-parsing"] }
 hmac = { version = "0.12.1", default-features = false, features = ["reset"] }
```

Comment:
this is another "i just whacked it with a hammer until it built again" - i think this is a result of patching `progenitor` like i did, and will probably not be necessary (or will at least be more explicable) once i've gone and bumped versions. this does not go in as-is.